index.html

```diff
@@ -466,7 +467,7 @@ <h2 class="contact-me-header-desktop">
             name="message"
           ></textarea>
           <button
-          id="submitBtn"
+            id="submitBtn"
```

Comment:
Your form section looks a bit different from the given design😉. Kindly consider the spacing around the highlighted text in the image below and make the changes where necessary👍.

yours
<img width="328" alt="image" src="https://github.com/GeekyHacks/My-Personal-Portfolio/assets/64643995/41a7816e-6169-4f94-8cff-85f3be0f01fe">

design
<img width="268" alt="image" src="https://github.com/GeekyHacks/My-Personal-Portfolio/assets/64643995/074a46cc-c578-46d8-a24e-d14016802c8c">